db/dcrpg/pgblockchain.go

```diff
@@ -2364,15 +1983,17 @@ func (pgb *ChainDB) UpdateSpendingInfoInAllAddresses() (int64, error) {
 	}
 
 	updatesPerDBTx := 500
+	totalVinIbIDs := len(allVinDbIDs)
 
-	log.Infof("Updating spending tx info for %d addresses...", len(allVinDbIDs))
+	var timeStart time.Time
```

Comment:
`timeStart` needs a non-zero initial value so the first call to `time.Since` isn't crazy.  Either that or move the `timeStart = time.Now()` in the loop before `time.Since`.